docs/Project-Configuration/Project-Configuration-Files.md

```diff
@@ -392,16 +392,19 @@ The validation step will check for:
 
 ### Modules
 
-For patches that run tests based off of changes across multiple
+For versions that run tests based off of changes across multiple
 projects, the modules field may be defined to specify other git projects
 with configurations specifying the way that changes across them are
 applied within the patch at runtime. If configured correctly, the left
 hand side of the Spruce UI under "Version Manifest" will contain
 details on how the modules were parsed from YAML and which git revisions
 are being used.
 
+For mainline commits and [trigger versions](Project-and-Distro-Settings.md#project-triggers), a new 
+manifest will be created that uses the latest revision available for each module.
+
 For manual patches and GitHub PRs, by default, the git revisions in the
-version manifest will be inherited from its base version. You can change
+version manifest will be inherited from its base version (i.e. the most recent mainline commit version). You can change
```

Comment:
Do GitHub PR patches and manual patches always use the most recent mainline commit version as the base? I thought they used whatever is the base commit of the PR or the base commit in the local git repo for manual patches, both of which could be something other than the most recent commit.

Reply:
Good catch - that's correct